### ISSUE TITLE
New mail notification shown multiple times

### ISSUE THREAD

**Issue by antalm** [KeZhang/outlook_email_extensions#1]
Once a new mail arrives a new mail notification is shown every 30 seconds. Moreover, if you don't mark the e-mail as read for a long time, a new mail notification will be shown hundreds of times even if you already marked the e-mail as read.

**Comment by antalm** [comment]
I made the fix. If anyone else has the same problem, here's how to fix it:

1) create content_script.js with the following content:
```
var prevNewCount = 0;

function sendNotification(msg, count) {
  if (Notification.permission !== "granted")
    Notification.requestPermission();
  else {
    msg && new Notification(msg + '  ' + new Date());
  }
  chrome.runtime && chrome.runtime.sendMessage({"count": count});
}

setInterval(function() {
  var inbox = $('[title="Inbox"] + div')[0];
  var newCount = parseInt($(inbox).text());
  if (!isNaN(newCount) && newCount > 0 && newCount != prevNewCount) {
    sendNotification(newCount + ' New Email Unread', newCount);
  } else {
    sendNotification(null, 0);
  }
  prevNewCount = newCount;
}, 10 * 1000)
```
2) overwrite the following file with the above created content_script.js:

On Linux:
```~/.config/google-chrome/Default/Extensions/<random letters>/0.2.1_0/content_script.js```

On Windows: 
```C:\Documents and Settings\<username>\Local Settings\Application Data\Google\Chrome\User Data\Default\Extensions\<random letters>\0.2.1_0\content_script.js```

3) open chrome://extensions URL in Chrome

4) enable Developer mode

5) click Load unpacked extensions...

6) select the extensions's directory and click Open

On Linux:
```~/.config/google-chrome/Default/Extensions/<random letters>/0.2.1_0```

On Windows:
```C:\Documents and Settings\<username>\Local Settings\Application Data\Google\Chrome\User Data\Default\Extensions\<random letters>\0.2.1_0```

Now you will have the notification shown only once per new e-mail, and it will disappear after 20 seconds.

7) restart Chrome

Ke Zhang: if you like my solution above, please add it to the next release.

**Comment by KeZhang** [comment]
@antalm  Thank you for request.  And I have fixed this issue on v0.3.0
